src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -8,14 +8,19 @@
 
 public class SelectTestFigureOptionListener implements ActionListener {
 
-	private DriverManager driverManager;
+    private DriverManager driverManager;
 
-	public SelectTestFigureOptionListener(DriverManager driverManager) {
-		this.driverManager = driverManager;
-	}
+    public SelectTestFigureOptionListener(DriverManager driverManager) {
+        this.driverManager = driverManager;
+    }
 
-	@Override
-	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
-	}
+    @Override public void actionPerformed(ActionEvent e) {
+        if(e.getActionCommand().equals("Figure Joe 1")){
+            FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+        }else{
+            FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+        }
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.